packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -0,0 +1,36 @@
+---
+id: Right-to-left handbook
+title: PatternFly right-to-left handbook
```

Comment:
While the main focus is RTL, we do touch on stuff not specific to RTL on this page; wondering if something like "Bidrection(ality) handbook" would be a more fitting title?

Reply:
While I agree that it's more technically correct, I also think that most people would be more likely to find the information if we call it right-to-left - wdyt, @edonehoo? 

On this and the other wording changes - I defer to @edonehoo and if you want to merge and then Erin you add/change whatever you'd like, that's totally fine with me - I really just wanted to get the technical details in and get this documentation started.

Reply:
Yeah that makes sense regarding the naming. I'm good with it as-is.

And I'd be good with a followup to tweak things further if need be; this is definitely in a good spot to get up with the rest of the RTL updates

Reply:
I think I left that as an idea somewhere within my many edits - I also thought "bidirectionality handbook" was more accurate. Or "bidirectional language handbook/support"? But I also agree that it's more nitpicky and definitely not blocking getting this information live